### ISSUE TITLE
No GetVar( ) method for Urho.Node class

### ISSUE THREAD

**Issue by laheller** [xamarin/urho#338]
The _Urho.Node.SetVar(key,value)_ method looks a bit useless, since there is **no** _Urho.Node.GetVar(key)_ method.
Or at least the last available nuget package version (1.8.93) of UrhoSharp does not provide GetVar method for Urho.Node class.
Maybe I am wrong but what is then then the purpose of method SetVariable while we cannot get the value of variable?

**Comment by OmidID** [comment]
It's looks generator missing method. I'll take a looks.

**Comment by OmidID** [comment]
@laheller That should be generate because the method you used `Node_GetVar11` was changed to `Node_GetVar_11` https://github.com/xamarin/urho/blob/master/Bindings/Portable/Generated/binding.cpp#L6576

So on next version your app will be crash.

**Comment by laheller** [comment]
@OmidID the above is just a workaround for current v1.8.93 version.

**Comment by OmidID** [comment]
@laheller Yes i understand. that method you used it's generated. so that why it's should be fix by generator.
